### ISSUE TITLE
Consistency of death tiles regarding Race

### ISSUE THREAD

**Issue by Zatline** [teeworlds/teeworlds#3056]
Currently, a lot of death tiling in Race is being done with the use of teleportation tiles which are either used to teleport a player back to a checkpoint or are used to teleport the player into a kill box that ends the player.

So you can imagine regarding #2789 there is a major consistency issue in relation to vanilla as death tiles usually are a lot more sensitive than teleportation tiles. It’s probably for that reason why over all the years the community adopted the use of teleportation tiles universally as it is a lot more forgiving and makes runs consistent within themself.

I see two ways to address this;

- reduce the sensitivity of death tiles to more or less mimic teleportation tiles 
- increase the sensitivity of teleportation tiles to mimic that of death tiles

However, the introduction of teleportation tiles deserves a whole other issue in my opinion, as teleportation tiles would greatly benefit from being somewhat visually recognizable and by playing a sound effect upon using, which then again makes teleports not good for the use of ending players in general so 2nd solution already creates new issues … 
-> Meaning that we should probably opt for the use of death tiles with a universally reduced sensitivity and more importantly the ability to point towards checkpoints / specific spawns.

**Comment by AssassinTee** [comment]
@Zatline first, minor corrections: you mean size and not sensitivity, teleporters have smaller hitboxes and are therefore smaller. The middle of your Tee has to be within the tile (which is actually bad practice, doesn't account for sub ticks, etc.)

> reduce the sensitivity of death tiles to more or less mimic teleportation tiles
Please don't, deathtiles are perfectly fine in their size

> by playing a sound effect upon using
Please don't, that would be the first sound I would remove from the game because you will hear it over and over again and again

> and more importantly the ability to point towards checkpoints / specific spawns.
Sounds like a teleporter to me 😆 

I suggest simply adding the existing teleporters as a follow up on #2790 and fix the very minor issue of sub-tick-teleportation.
With that you could introduce another type of teleporter which has the size of a deathtile or even bigger and support other stuff like projectile teleportation

**Comment by fokkonaut** [comment]
Just keep it as it is? Its perfect like this and has all possibilities. I see no problem in using teleporters to teleport into a killtile.

**Comment by Zatline** [comment]
@AssassinTee  
> you mean size and not sensitivity, teleporters have smaller hitboxes and are therefore smaller.

yes I do mean hitbox thanks, wrote this a bit too early in the morning xD

>Please don't, deathtiles are perfectly fine in their size

hmm, I always felt like regular death tiles are a bit too big and punishing, I always preferred teleportation tiles "acting" as death tiles as they have a smaller hitbox thus are more forgiving.

> Please don't, that would be the first sound I would remove from the game because you will hear it over and over again and again

well, that's exactly why I wouldn't want true teleportation tiles to be used as deathtiles, it would get annoying very quick.

> and more importantly the ability to point towards checkpoints / specific spawns.

well, technically it would but death should feel like death and not straight-up teleportation that's the point.
It's not consistent when in vanilla you touch a death tile and immediately die while in race you can touch it slightly due to tele tiles smaller hitbox .. there shouldn't be a difference! 
So either make one smaller or the other bigger or do both for each and meet in the middle.

@fokkonaut 

> I see no problem in using teleporters to teleport into a killtile.

I don't have a problem either only that this is an unnecessary step that could be skipped if the death tiles hitbox were consistent with teleportation tiles or vice versa.

**Comment by ChillerDragon** [comment]
> reduce the hitbox of death tiles to more or less mimic teleportation tiles

Yea please don't :c

**Comment by oy** [comment]
Don't think that the hitbox of death tiles should be changed. People are used to them. 
And I guess people are used to the behaviour of teleport tiles as well.
